site/main.css

```diff
@@ -18,55 +80,93 @@ table
     max-width: 100%;
 }
 
+html {
+    font-size: 62.5%;
+}
 
 body
 {
     margin: 0;
-    line-height: 1.4;
+    line-height: 1.5;
+    font-size: 1.6rem;
 }
 
 /** Structural blocks */
 
 #page-wrapper
 {
-    max-width: 1000px;
+    max-width: 88rem;
+
+    padding: 1rem 2rem 2rem;
 
-    padding-top: 2%;
-    padding-bottom: 4%;
-    
     margin-left:  auto; 
     margin-right: auto;
 }
 
+
 /* Visual styles */
-header 
+html
 {
-    text-align: center;
+    background: #fff;
+    background: var(--Canvas-bg, #fff);
+    color: #000;
+    color: var(--Canvas-color, #000);
 }
 
-nav
+header
 {
-    font-size: 110%;
-
-    border-bottom: 2px solid #c90309;
+    text-align: center;
+}
 
-    padding-top: 5px;
-    padding-bottom: 5px;
+/* shared gradient border */
+nav, footer 
+{
+    border-width: 0;
+    border-style: solid;
+    border-color:  #c90309;
+    border-color: var(--Accent1-color, #c90309);
+    border-image: linear-gradient(to right, var(--Accent1-color, #c90309), var(--Accent2-color, #d65906)) 2;
+}
 
+nav
+{
+    font-size: 1.8rem;
+    border-bottom-width: 2px;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
     text-align: center;
 }
 
+main
+{
+    color: #222;
+    color: var(--Main-color, #222);
+    text-rendering: optimizeLegibility;
```

Comment:
This is a non-standard property that seems to have no real effect in any modern browser. Why add it?

Reply:
> One very visible effect is optimizeLegibility, which enables ligatures (ff, fi, fl, etc.) in text smaller than 20px for some fonts (for example, Microsoft's Calibri, Candara, Constantia, and Corbel, or the DejaVu font family).

https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering

It looks like it depends on what your system font is?